src/Libraries/CoreNodesUI/Input/SliderBase.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace DSCoreNodesUI.Input
+{
+    public abstract class SliderBase<T> : BasicInteractive<T>
+    {
```

Comment:
This can be achieved by adding the constraint `where T: IComparable<T>` But, because the IComparable interface knows nothing about the operators `< > <= >=`, you need to use the interface's CompareTo() method. Works well enough.